cmd/install/install_render.go

```diff
@@ -165,22 +192,100 @@ func hyperShiftOperatorTemplateManifest(opts *Options) (crclient.Object, error)
 	}
 	templateParameters = append(
 		templateParameters,
-		map[string]string{"name": TemplateParamOperatorReplicas, "value": "1"},
+		map[string]interface{}{"name": TemplateParamOperatorReplicas, "value": "1", "required": true},
 	)
 	patchedObjects, err := applyPatchesToObjects(objects, patches)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	return patchedObjects, templateParameters, nil
+}
+
+func hyperShiftOperatorTemplateManifest(opts *Options) (crclient.Object, error) {
+	objects, templateParameters, err := hyperShiftOperatorTemplateObjects(opts)
+	if err != nil {
+		return nil, err
+	}
+	// wrap into template
+	template := &unstructured.Unstructured{
+		Object: map[string]interface{}{
+			"kind":       "Template",
+			"apiVersion": "v1",
+			"metadata": map[string]string{
+				"name": "hypershift-operator-template",
+			},
+			"objects":    objects,
+			"parameters": templateParameters,
+		},
+	}
+	return template, nil
+}
+
+func hyperShiftOperatorSSSTemplateManifest(opts *Options) (crclient.Object, error) {
+	objects, templateParameters, err := hyperShiftOperatorTemplateObjects(opts)
 	if err != nil {
 		return nil, err
 	}
 
+	// wrap into sss
+	sss := map[string]interface{}{
+		"kind":       "SelectorSyncSet",
+		"apiVersion": "hive.openshift.io/v1",
+		"metadata": map[string]interface{}{
+			"name": fmt.Sprintf("hypershift-operator-${%s}-${%s}", SSSTemplateParamEnvironment, SSSTemplateParamSectorValue),
+			"annotations": map[string]string{
+				"kubernetes.io/description": "Deploys hypershift-operator on the selected environment/sector management clusters",
+			},
+		},
+		"spec": map[string]interface{}{
+			"clusterDeploymentSelector": map[string]interface{}{
+				"matchLabels": map[string]interface{}{
+					"api.openshift.com/managed": "true",
+				},
+				"matchExpressions": []map[string]interface{}{{
+					"key":      "api.openshift.com/fedramp",
+					"operator": "NotIn",
+					"values":   "true",
+				}, {
+					"key":      "api.openshift.com/environment",
+					"operator": "In",
+					"values":   []string{fmt.Sprintf("${%s}", SSSTemplateParamEnvironment)},
+				}, {
+					"key":      fmt.Sprintf("${%s}", SSSTemplateParamManagementClusterKey),
+					"operator": fmt.Sprintf("${%s}", SSSTemplateParamManagementClusterOperator),
+					"values":   []string{fmt.Sprintf("${%s}", SSSTemplateParamManagementClusterValue)},
+				}, {
+					"key":      fmt.Sprintf("${%s}", SSSTemplateParamSectorKey),
+					"operator": fmt.Sprintf("${%s}", SSSTemplateParamSectorOperator),
+					"values":   []string{fmt.Sprintf("${%s}", SSSTemplateParamSectorValue)},
+				}},
+			},
+			"resourceApplyMode": "Sync",
+			"resources":         objects,
+		},
+	}
+
+	templateParameters = append(
+		templateParameters,
+		map[string]interface{}{"name": SSSTemplateParamEnvironment, "value": SSSTemplateParamDefaultEnvironment, "required": true},
```

Comment:
nit: we could let the option param in `templatize` func be a plain string and use it here for all this.